### PR TITLE
Update docker compose references to support working in CI and python3

### PR DESCRIPTION
**Pull request by unlikelyzero**
Addresses https://github.com/yamcs/quickstart/issues/11 and https://github.com/yamcs/quickstart/issues/9

### CODE REVIEW

**Comment by fqqb** [comment]
@unlikelyzero to me appears to look good, so just indicate when you are done and i'll merge

**Comment by unlikelyzero** [comment]
@fqqb this should be good to go! Would you mind squashing the commits?

File: pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     Update this to the latest Yamcs release.
     Check https://mvnrepository.com/artifact/org.yamcs/yamcs-core
     -->
-    <yamcsVersion>5.7.7</yamcsVersion>
+    <yamcsVersion>5.7.8</yamcsVersion>
```
Comment (unlikelyzero) [inline pom.xml:19]: Drive-by: Updated to latest while i was in here

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -5,8 +5,21 @@ services:
     hostname: yamcs
     container_name: yamcs
     command: "mvn yamcs:run"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8090"]
+      interval: 10s
+      timeout: 5s
+      retries: 30
+      start_period: 30s
```
Comment (unlikelyzero) [inline docker/docker-compose.yml:13]: healthcheck for docker compose
